javaServices/coreJavaServices/src/joliex/io/FileService.java

```diff
@@ -744,4 +745,38 @@ public boolean accept( File directory, String filename )
 			return pattern.matcher( filename ).matches() && (!dirsOnly || file.isDirectory());
 		}
 	}
+
+	@RequestResponse
+	public Value toAbsolutePath( Value request ) throws FaultException
+	{
+		Value response = Value.create();
+		String fileName = request.strValue();
+		
+		String absolutePath = Paths.get( fileName ).toAbsolutePath().normalize().toString();
+
+		if ( absolutePath == null ) {
+			throw new FaultException( "Not a valid path", new IOException() );
```

Comment:
Should be

throw new FaultException( new IOException( "Not a valid path" ) );

Same for the other method.